.yamllint

```diff
@@ -2,7 +2,6 @@ rules:
   document-start:
     level: error
   line-length: disable
-  key-duplicates: disable
```

Comment:
I think this needs to be `key-duplicates: {}` or `key-duplicates: enable` instead of nothing given we don't extend the default in this config.

Reply:
Can switch to extending default now, no?

Reply:
You can give it a go if you want

Reply:
At least the key duplicate errors have proved me right 🤣